README.md

```diff
@@ -90,6 +90,21 @@ See the
 [scan result object](https://github.com/sodadata/soda-sql/blob/main/core/sodasql/scan/scan_result.py)
 for all attributes and methods.
 
+Or, return Spark data frames:
+
+``` python
+>>> measurements, test_results, errors = scan.execute(scan_yml, df, as_frames=True)
+>>>
+>>> measurements  # doctest: +ELLIPSIS
+DataFrame[metric: string, column_name: string, value: string, ...]
```

Comment:
This doc example is also a test(!). This is done with [doctest](https://docs.python.org/3/library/doctest.html)